src/main/java/jenkins/plugins/publish_over_ssh/BapSshTransfer.java

```diff
@@ -70,26 +73,37 @@ public boolean isUsePty() {
         return usePty;
     }
 
+    public boolean isUseAgentForwarding() {
+        return useAgentForwarding;
+    }
+
+    @DataBoundSetter
+    public void setUseAgentForwarding(boolean value) {
```

Comment:
I would add `@Restricted(NoExternalUse)` this method is not supposed to be used in binary API